### PR TITLE
Update connecting-your-database.md

### PR DESCRIPTION
**Pull request by mirnawong1**
Added a note that the schema field is a required field in dev credentials

**Note**: Heads up! The schema field in the 'Developer Credentials' section is a required field.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ Development or Deployment credentials in a dbt project. Simply enter your Snowfl
 username (specifically, the `login_name`) and the corresponding user's Snowflake `password`
 to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowflake user.
 
+**Tip**: The schema field in the 'Developer Credentials' section is a required field.
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: nitpick with the word `tip`. doesn't that mean something that helps someone work more efficiently? in this case it doesn't strike me as a tip but a requirement...

Comment (mirnawong1) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: Good flag, thank you @dataders ! i'll change it to note

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 2]
```diff
@@ -69,6 +69,8 @@ Development or Deployment credentials in a dbt project. Simply enter your Snowfl
 username (specifically, the `login_name`) and the corresponding user's Snowflake `password`
 to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowflake user.
 
+**Note**: The schema field in the 'Developer Credentials' section is a required field.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
**Note**: The schema field in the **Developer Credentials** section is a required field.
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: it should be bold if it's a UI element (ie these words can be seen in the GUI). if not, leave it as is

**Review by nghi-ly — COMMENTED** [review]
left one comment

**Comment by mirnawong1** [comment]
thanks @nghi-ly , committed those changes and let me know when i can go ahead and merge 🙏

**Comment by mirnawong1** [comment]
@mirnawong1

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**